lib/l1_sender/src/pipeline_component.rs

```diff
@@ -27,7 +27,7 @@ where
     type Output = BatchEnvelope<FriProof>;
 
     const NAME: &'static str = C::NAME;
-    const OUTPUT_BUFFER_SIZE: usize = 5;
+    const OUTPUT_BUFFER_SIZE: usize = 1;
```

Comment:
Why `1`?

Reply:
I will revisit all buffer sizes to be more delibarate. Just putting 5 everywhere is not a good approach - e.g. gapless committer itself shouldn't have any input buffer (it has a buffer inside). 

FYI this one I set to `1` now because I added a buffer in SnarkProverPipelineStep (there is a comment there).